examples/cmri_mpca/config.py

```diff
@@ -0,0 +1,77 @@
+"""
+Default configurations for CMR processing and classification
+"""
+
+import os
+
+import numpy as np
+import pydicom
+from yacs.config import CfgNode as CN
+
+# -----------------------------------------------------------------------------
+# Config definition
+# -----------------------------------------------------------------------------
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.SOURCE = "https://github.com/pykale/data/raw/main/image_data/ShefPAH-179/SA_64x64.zip"
+_C.DATASET.ROOT = "../data"  # '/shared/tale2/Shared'
+_C.DATASET.IMG_DIR = "DICOM"
+_C.DATASET.FILENAME = "PAH_SA_64x64"
```

Comment:
Are you sure this is still the *file* name? Which *file*?
File structure: why SA_64x64/PAH_SA_64x64/...? Will there be another dir SA_64x64?